package.json

```
@@ -19,7 +19,8 @@
     "elements",
     "lib",
     "src",
-    "RNSVG.podspec"
+    "RNSVG.podspec",
+    "!build"
```

Isn't it too generic now? Maybe something like this would be more reasonable:

```suggestion
    "!android/build"
```